cmd/commands/repo.go

```diff
@@ -335,13 +344,78 @@ func NewRepoUninstallCommand() *cobra.Command {
 	return cmd
 }
 
+func RemoveFromRepo(ctx context.Context, opts *RepoUninstallOptions) (git.Repository, fs.FS, string, error) {
```

Comment:
now this func doesn't need to check for `opts.Force` anymore. Just return the errors if they happen.
the calling function can check for `opts.Force` and decide what to do there.
same for all the similar funcs that uninstall calls.